.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: "\U0001F41B Bug report"
+about: Create a report to help us improve
+
+---
+<!-- Issues are for bugs and feature requests** only -->
```

Comment:
What do the `**` mean?

Reply:
It's bold. I forget "**" in the beginning

Reply:
I forget twice * in the beginning. It's a bold comment